kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -141,7 +139,10 @@
   import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
   import CoachContentLabel from 'kolibri.coreVue.components.CoachContentLabel';
   import DownloadButton from 'kolibri.coreVue.components.DownloadButton';
-  import { isEmbeddedWebView, isAndroidWebView } from 'kolibri.utils.browser';
+  import client from 'kolibri.client';
+  import urls from 'kolibri.urls';
+  import { isEmbeddedWebView, isAppView } from 'kolibri.utils.browser';
+  import { launchIntent } from 'kolibri.utils.appCapabilities';
```

Comment:
To the note above about what happens if a capability isn't defined, I'm assuming this would error if it's not defined?

Reply:
It would just be 'undefined'.